lib/report_parser.ex

```diff
@@ -34,29 +61,38 @@ defmodule RecordingConverter.ReportParser do
     |> Jason.decode!()
   end
 
+  defp get_video_tracks_offset(tracks) do
+    tracks
+    |> Enum.filter(&(&1["type"] == "video"))
+    |> Enum.reduce(%{}, fn %{"origin" => origin, "offset" => offset}, acc ->
+      Map.update(acc, origin, [offset], &[offset | &1])
+    end)
+    |> Map.new(fn {origin, offset} -> {origin, Enum.sort(offset)} end)
+  end
```

Comment:
Won't there be a problem when somebody has two video tracks, e.g: his camera video and screen share? Because screen share tracks also have type video [if I remember correctly](https://github.com/jellyfish-dev/membrane_rtc_engine/blob/a71f7642c82e9c8c455f475d9cfd2abaf9dcef96/engine/lib/membrane_rtc_engine/track.ex#L67).